.circleci/conditional_config.yml

```diff
@@ -49,11 +49,19 @@ images:
     image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base:latest
     <<: *common_frontend_config
 
+  - &frontend_ci
+    image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base-ci:latest
```

Comment:
new image 600 MB smaller than `frontend-base`

Reply:
The image is called `frontend-base-ci` on the registry, but it's called `frontend-ci` here. I suggest keeping the same name here.

Reply:
would be ok for you @hennevogel ?

Reply:
What exactly? Using the same name? For sure.